tests/WorkflowTest.php

```diff
@@ -254,6 +254,30 @@
     assertEquals(0, $_SERVER['__catch.count']);
 });
 
+it('has ran by default', function () {
```

Comment:
This test name is a little misleading. Because a workflow has _not_ run by default. It's only true in this specific case because the `createWorkflow()` helper function creates a workflow without any jobs by default. You probably got misled by some of the other tests in this file that follow a similar naming convention.

Can you change this test to `has not run by default` and adjust the test setup so that the workflow has at least one job?

Reply:
Working on it!